### PR TITLE
Assignment4

### PR DESCRIPTION
**Pull request by Shelvi96**

### CODE REVIEW

File: Assignment4/Shelvi96-XXXXXX/task07.py
Comment (anaigmo) [inline Assignment4/Shelvi96-XXXXXX/task07.py:1]: missing rdflib alternatives for all exercises in this task

File: Assignment4/Shelvi96-XXXXXX/task07.py [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+FOAF = Namespace("http://xmlns.com/foaf/0.1/")
+VCARD = Namespace("http://www.w3.org/2001/vcard-rdf/3.0#")
+
+
+"""**TASK 7.1: List all subclasses of "LivingThing" with RDFLib and SPARQL**"""
+
+q1 = prepareQuery('''
+    SELECT ?SubClass WHERE { 
+        ?SubClass rdfs:subClassOf ns:LivingThing.
+    }
```
Comment (anaigmo) [inline Assignment4/Shelvi96-XXXXXX/task07.py:38]: missing recursive subclassof relationships

File: Assignment4/Shelvi96-XXXXXX/task07.py [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+    print(ns)
+
+"""**TASK 7.4:  List the name of the persons who know Rocky**"""
+
+q4 = prepareQuery('''
+    SELECT ?individual
+    WHERE {
+        ?individual rdf:type ns:Person.
+        ?individual foaf:knows ns:RockySmith.
+    }
```
Comment (anaigmo) [inline Assignment4/Shelvi96-XXXXXX/task07.py:89]: list name, not iri

Comment (Shelvi96) [reply Assignment4/Shelvi96-XXXXXX/task07.py]: done

File: Assignment4/Shelvi96-XXXXXX/task07.py (hunk 1 above)
Comment (Shelvi96) [reply Assignment4/Shelvi96-XXXXXX/task07.py]: done

**Comment by dgarijo** [comment]
Still missing all exercises in rdflib

**Comment by Shelvi96** [comment]
done

File: Assignment4/Shelvi96-XXXXXX/task07.py
Comment (Shelvi96) [reply Assignment4/Shelvi96-XXXXXX/task07.py:1]: done

**Review by dgarijo — APPROVED** [review]

**Pull request merged**